aspnet/web-forms/overview/moving-to-aspnet-20/improvements-in-visual-studio-2005.md

```diff
@@ -56,7 +56,7 @@ Web developers have many new options available for project creation in Visual St
 
 Click here to view a video walkthrough of creating Web projects in Visual Studio 2005.
 
-![](improvements-in-visual-studio-2005/_static/image1.png)
+![Screenshot of the video walkthrough of creating Web projects in Visual Studio.](improvements-in-visual-studio-2005/_static/image1.png)
```

Comment:
I don't think this is a screenshot, isn't this more of the icon that they click to start the video?